logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -198,7 +198,44 @@ actual class CallManagerImpl(
             callingLogger.d("$TAG - onCallingMessageReceived")
         }
 
-    override suspend fun startCall(conversationId: ConversationId, callType: CallType, conversationType: ConversationType, isAudioCbr: Boolean) {
+    private fun sendHandlerSuccess(
+        messageString: String?,
+        conversationId: ConversationId,
+        avsSelfUserId: UserId,
+        avsSelfClientId: ClientId
+    ) {
+        scope.launch {
+            messageString?.let { message ->
+                withCalling {
+                    when (sendCallingMessage(conversationId, avsSelfUserId, avsSelfClientId, message)) {
+                        is Either.Right -> {
+                            wcall_resp(
+                                inst = deferredHandle.await(),
+                                status = 200,
+                                reason = "",
+                                arg = null
```

Comment:
We need to pass something here not null, we get it from sendHandler (first parameter)